client/landing/subscriptions/components/tab-views/comments/comments.tsx

```diff
@@ -1,13 +1,15 @@
 import { SubscriptionManager, Reader } from '@automattic/data-stores';
+import { SiteSubscriptionsFilterBy } from '@automattic/data-stores/src/reader/queries';
```

Comment:
```suggestion
```
eslint error: 'SiteSubscriptionsFilterBy' is defined but never used. (@typescript-eslint/no-unused-vars)
We can remove this.